sycl/include/sycl/handler.hpp

```diff
@@ -1137,6 +1164,7 @@ class __SYCL_EXPORT handler {
     kernel_parallel_for_work_group_wrapper<NameT, LambdaArgType, KernelType,
                                            PropertiesT>(KernelFunc);
 #ifndef __SYCL_DEVICE_ONLY__
+    processProperties<PropertiesT>();
```

Comment:
If you move this out of the `#ifndef __SYCL_DEVICE_ONLY__` I believe you can remove the same call inside the `_unpack` functions as they are only called through the `*_wrapper` calls above.

That is, assuming we need this for device too, which I don't believe we do, so maybe they can just be removed from the `*_unpack` calls?

Reply:
I moved properties processing from *_unpack calls to unpack() function: https://github.com/intel/llvm/pull/8597/files#diff-352c77f00d97af3d3595fc337c97c0586ca3455500738cae7836acb22bb6d282R1359-R1365

Because it is the place where we currently take care of proprties returned by get(properties_tag) method.
And  this is also the reason why this code is needed there because otherwise properties returned by get(properties_tag) will be ignored.